samples/sample-04/src/main/resources/application.yml

```diff
@@ -0,0 +1,30 @@
+logging:
+  level:
+    org:
+      springframework: DEBUG
+      hibernate:
+        SQL: debug
+spring:
+  datasource:
+    url: jdbc:mysql://localhost:3306/notes
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    username: root
+    password: null
+  jpa:
+    database: MYSQL
+    database-platform: org.hibernate.dialect.MySQL5Dialect
+    show-sql: true
+    open-in-view: true
+    hibernate:
+      ddl-auto: update
+    repositories:
+      bootstrap-mode: default
+  kafka:
+    bootstrap-servers: localhost:9092
+    consumer:
+      group-id: noteGroup
+      auto-offset-reset: earliest
+      properties:
+        spring:
+          json:
+            trusted:packages: com.remusrd.notesample.domain.event
```

Comment:
What is this package?
Doesn't look like legitimate package name in the public project.